x-pack/test/api_integration/apis/secops/kpi_network.ts

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import expect from 'expect.js';
+import { kpiNetworkQuery } from '../../../../plugins/secops/public/containers/kpi_network/index.gql_query';
+import { GetKpiNetworkQuery } from '../../../../plugins/secops/public/graphql/types';
+import { KbnTestProvider } from './types';
+
+const kpiNetworkTests: KbnTestProvider = ({ getService }) => {
+  const esArchiver = getService('esArchiver');
+  const client = getService('secOpsGraphQLClient');
+  describe('Kpi Network', () => {
+    describe('With filebeat', () => {
+      before(() => esArchiver.load('filebeat/default'));
+      after(() => esArchiver.unload('filebeat/default'));
+
+      const FROM = new Date('2019-02-09T01:57:24.870Z').valueOf();
+      const TO = new Date('2019-02-12T01:57:24.870Z').valueOf();
```

Comment:
You can keep these ranges to whatever you feel is the most important, but I do want to point out what I found from running the `esArchiver` after Beats updates a few times now is that I prefer to keep my `FROM` and `TO` as wide of a net as possible to try to reduce the amount of times the integration tests break.

So, for example, in a lot of the integration tests I prefer to do:

```ts
const FROM = new Date('2000-01-01T00:00:00.000Z').valueOf();
const TO = new Date('3000-01-01T00:00:00.000Z').valueOf();
```

So that the next time I run the EsArchiver I might have the best chance of getting all the record sets again. Each time we run them it is usually on a newer data set that won't have the data set of `2019-02-12T01:57:24.870Z` in  it for example.

This is fine as is, just pointing out, I try to get all the record sets these days to reduce the maintenance of when I have to re-run esarchiver.

Thanks for following the other patterns as this will still be easy to change as we update archives.